src/js/const/img-urls.const.js

```diff
@@ -0,0 +1,4 @@
+export const IMG_URL = {
```

Comment:
`ASSET_SUBTYPE_IMG_URL`

Reply:
And you can move it to the `src/js/const/asset-subtypes.const.js`

Reply:
ok